### ISSUE TITLE
Could I scale for 2 instances?

### ISSUE THREAD

**Issue by cmdntd** [alexandre-abrioux/golem-node#29]
Could I scale golem for 2 instances or more?

```
.PHONY: up
up: 			## Start the node
	docker-compose up -d --remove-orphans
```
to
```
.PHONY: up
up: 			## Start the node
	docker-compose up -d --remove-orphans --scale 5
```

**Comment by alexandre-abrioux** [comment]
Hi! You can probably have a setup with multiple nodes working in parallel, but I doubt this will work. Nodes will probably need to have different configurations, meaning:
- be attached to different volumes (different `./data-node/` per node)
- and maybe also be linked to different Ethereum wallet addresses

It's just wild guess, as I've never tried it. I'd advise contacting Golem's team [on their Discord](https://discord.gg/golem) for proper advice. Feel free to share your `docker-compose.yaml` file if you manage to do it! I'm closing this for now as it's a bit out of scope for this project.
